doc/admin/hardware.md

```diff
@@ -41,7 +41,7 @@ systems for ARM platforms.
 ## Network Requirements
 
 The DAOS network layer relies on libfabrics and supports OFI providers
-for Ethernet/sockets, InfiniBand/verbs, RoCE, Cray GNI, and Intel
+for Ethernet/sockets, InfiniBand/verbs, RoCE, and Intel
 Omni-Path Architecture (OPA). An RDMA-capable fabric is preferred for better
```

Comment:
unrelated to the original PR, but do you want to remove Omni-Path here?

Reply:
Yes, we should IMO.